core/textile/hub/hub_auth.go

```diff
@@ -63,26 +68,27 @@ func storeHubToken(st store.Store, hubToken string) error {
 	return err
 }
 
-func GetHubToken(ctx context.Context, st store.Store, kc keychain.Keychain, cfg config.Config) (string, error) {
+func GetHubToken(ctx context.Context, st store.Store, kc keychain.Keychain, cfg config.Config) (context.Context, error) {
 	// Try to avoid redoing challenge if we already have the token
-	if valFromStore, err := getHubTokenFromStore(st); err != nil {
-		return "", err
-	} else if valFromStore != "" {
-		log.Debug("Got hub token from store: " + valFromStore)
-		return valFromStore, nil
-	}
+	// if valFromStore, err := getHubTokenFromStore(st); err != nil {
+	// 	return nil, err
+	// } else if valFromStore != "" {
+	// 	log.Debug("Got hub token from store: " + valFromStore)
```

Comment:
We should reenable this, and add an expiry check before returning so that we can refresh.